x/tss/genesis.go

```diff
@@ -11,6 +11,7 @@ import (
 // from the genesis state
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, g types.GenesisState) {
 	k.SetParams(ctx, g.Params)
+	k.SetGovernanceKey(ctx, g.GovernanceKey)
```

Comment:
If `g.GovernanceKey` is not allowed to be nil, we should validate it as part of https://github.com/axelarnetwork/axelar-core/blob/6ae6a6730e108d5bedfd2343b757a365cbae2055/x/tss/types/genesis.go#L17